README.md

```diff
@@ -1 +1,150 @@
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project">
+Hello , This is a HTML AND CSS simple project.</a>
+
+[Hello Microverse] is a simple buil up demo project.
+
+## 🛠 Built With 
+
+### Tech Stack 
+HTML
+CSS
+
+<!-- Features -->
+
+### Key Features 
+
+- **[Simple Site]**
+- **[h1 tag is used]**
+- **[Linter is used]**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:You have to clone this project to your local machine
+
+### Setup
+
+Clone this repository to your desired folder:git clone git@github.com:Elhambasir/Hello-Microverse.git
+
+### Install
+
+Install this project with: Any software which you are comfortable with
+
+### Usage
+
+To run the project, execute the following command: On your computer's keyboard press f5 or run it with live server
+
+### Run tests
+
+To run tests, run the following command: In visual studio code with live server plugin right click and run live server.
+
+### Deployment
+
+You can deploy this project using: It is not completed.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@AliShahSafdari](https://github.com/AliShahSafdari)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+ **Future Features**
+
+- [ ] **[Navbar]**
+- [ ] **[Footer]**
+- [ ] **[Good style]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+> Write a message to encourage readers to support your project
+
+If you like this project reach me on my contact
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank in advance
```

Comment:
Please consider either tagging them or naming the ones you are acknowledging to complete the requirements of this section. Currently, you can acknowledge Microverse.